tests/integration/suite/test_catalog.py

```diff
@@ -183,3 +197,37 @@ def test_template_version_links(admin_mc, admin_pc, custom_catalog,
     assert len(templates.data[0]['versionLinks']) == 2
     assert '1.6.0' in templates.data[0]['versionLinks']
     assert '1.6.2' in templates.data[0]['versionLinks']
+
+
+def test_relative_paths(admin_mc, admin_pc, remove_resource):
+    """ This test adds a catalog's index.yaml with a relative chart url
+    and ensures that rancher can resolve the relative url"""
+
+    client = admin_mc.client
+    catalogname = "cat-" + random_str()
+    # requires "/" at the end to pass url.Parsing
+    url = "https://raw.githubusercontent.com/daxmc99/chart-test/master/"
+    catalog = client.create_catalog(catalogName=catalogname, branch="master",
+                                    url=url)
+    remove_resource(catalog)
+
+    catalog = client.reload(catalog)
+    assert catalog['url'] == url
```

Comment:
I see you're using `wait_for_template_to_be_created` below but won't this assert fail if it's not created yet ?